core/templates/components/state-editor/state-interaction-editor/state-interaction-editor.component.html

```diff
@@ -6,14 +6,6 @@
       <h3 class="oppia-exp-interaction-card-header">
         Interaction <span *ngIf="getCurrentInteractionName()">( {{getCurrentInteractionName()}} )</span>
       </h3>
-      <i class="fa fa-caret-down"
-         *ngIf="!interactionEditorIsShown"
-         aria-hidden="true">
-      </i>
-      <i class="fa fa-caret-up"
-         *ngIf="interactionEditorIsShown"
-         aria-hidden="true">
-      </i>
```

Comment:
What happens when an interaction is present? I think you have completely removed the dropfown icon, instead of hiding it only when there are no interactions.

Reply:
I think the job is done when Interaction is added the expand/collapse icon will reappear